spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,9 +31,16 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:wixtestfiles) { "/tmp/spec/fixtures/wix/resources/windows/wix" }
 
       before do
         cur_plat.instance_eval(plat[:block])
+        FileUtils.mkdir_p(wixtestfiles)
+        FileUtils.cp_r("spec/fixtures/wix/resources/windows/wix", "#{wixtestfiles}/..")
```

Comment:
this was a little bit of voodoo to make rspec play nice. `/tmp/spec/fixtures/wix/resources/windows/wix` is an absolute path, and since I did a chdir in the before block I could not for the life of me get rspec to find those files. so I made an absolute path.